src/test/java/com/orbitz/consul/SessionClientTest.java

```diff
@@ -1,26 +1,44 @@
 package com.orbitz.consul;
```

Comment:
I think it'd be worthwhile to test providing no fields in session creation, as well as a test specifying all fields and asserting that they are set correctly.